docs/guide/config_back_office.md

```diff
@@ -110,3 +110,30 @@ ezpublish:
                     date_format: 'dd/mm/yyy'
                     time_format: 'hh:mm'
 ```
+
+## Content Tree
+
+With this configuration you can:
+
+- define configuration for a SiteAccess or a SiteAccess group
+- decide how many Content items are displayed in the tree
+- set maximum depth of expanded tree
+- hide Content Types
+
+```yaml
+ezpublish:
+    system:
+        admin_group: # any SiteAccess or SiteAccess group
+            content_tree_module:
+                load_more_limit: 15 # defines how many children will be shown after expanding parent
```

Comment:
Maybe move all comments to the lines above the relevant setting? In the rendering the lines are too long and hard to read.